patternlab/styleguide/source/_patterns/02-molecules/pagination.twig

```diff
@@ -1,23 +1,32 @@
 <div class="ma__pagination js-pagination">
-  <div class="ma__pagination__container">
-    <button
-      class="ma__pagination__prev js-pagination-prev"
-      type="button"
-      {% if pagination.prev.disabled %}
-        disabled
-      {% endif %}>{{ pagination.prev.text }}</button>
+  <nav class="ma__pagination__container" aria-label="Pagination Navigation">
+    <a 
+      class="ma__pagination__prev js-pagination-prev {% if pagination.prev.disabled %}disabled{% endif %}"
+      {% if pagination.prev.disabled %}aria-disabled="true" {% else %}
+      aria-label="Go to {{ pagination.prev.text }} page"{% endif %}
+      href="#">{{ pagination.prev.text }}</a>

```

Comment:
`<a>` should be using href, is it currently attaching onClick to `js-pagination-prev`?

Reply:
should be adding `role="button"`